comps/dataprep/src/opea_dataprep_microservice.py

```diff
@@ -55,19 +55,21 @@ async def ingest_files(
     chunk_overlap: int = Form(100),
     process_table: bool = Form(False),
     table_strategy: str = Form("fast"),
-    ingest_from_graphDB: bool = Form(False),
+    key_index_name: Optional[str] = Form(None),
 ):
     start = time.time()
 
+    if key_index_name:
+        # Set key_input_name to environment variable
+        os.environ["KEY_INDEX_NAME"] = key_index_name
+
     if logflag:
         logger.info(f"[ ingest ] files:{files}")
         logger.info(f"[ ingest ] link_list:{link_list}")
 
     try:
         # Use the loader to invoke the component
-        response = await loader.ingest_files(
-            files, link_list, chunk_size, chunk_overlap, process_table, table_strategy, ingest_from_graphDB
-        )
+        response = await loader.ingest_files(files, link_list, chunk_size, chunk_overlap, process_table, table_strategy)
```

Comment:
Wouldn't this affect other dataprep implementations? (for example [vdms](https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/src/integrations/vdms.py#L154))